tidb-cloud/setup-azure-self-hosted-kafka-private-link-service.md

```
@@ -441,7 +441,7 @@ b3.abc.eastus.azure.3199745.tidbcloud.com:9095 (id: 3 rack: null) -> ERROR: org.
     -   **IP version**: `IPv4`
     -   **Virtual network**： `kafka-pls-vnet`
     -   **サブネット**: `brokers-subnet`
-    -   **課題**： `Dynamic`
+    -   **Assignment**： `Dynamic`
```

markdownlint-cli2 (0.23.2)

[warning] Spaces after list markers
Expected: 1; Actual: 3

(MD030, list-marker-space)